Backend/tests/test_MentiiAuth.py

```diff
@@ -0,0 +1,85 @@
+import unittest
+from contextlib import contextmanager
+from flask import appcontext_pushed, g, Flask
+
+import db_utils as db
+from botocore.exceptions import ClientError
+
+import boto3
+import sys
+from os import path
+sys.path.append( path.dirname( path.dirname( path.abspath(__file__) ) ) )
+from mentii import user_ctrl
+from utils.MentiiAuth import MentiiAuthentication
+
+app = Flask(__name__)
+
+#local DynamoDB
+dynamodb = boto3.resource('dynamodb', endpoint_url='http://localhost:8000')
+
+class MentiiAuthTests(unittest.TestCase):
+
+  @classmethod
+  def setUpClass(self):
+    settingsName = "table_settings.json"
+    mockData = "mock_data.json"
+
+    try:
+      table = db.createTableFromFile("./tests/"+settingsName, dynamodb)
```

Comment:
Just for future reference I **_think_** this could be because the methods on MentiiAuth are accessed statically and an instance of it is never made for testing